packages/compiler/core/logger.browser.ts

```diff
@@ -0,0 +1,10 @@
+// noop logger shouldn't be used in browser
```

Comment:
This is a bit annoying to have to do that for every CLI specific functionality. Wondering if maybe we should have a `@cadl-lang/cli` library and leave `@cadl-lang/compiler` for all the programmatic usage.

But this is yet another library to have to install.

Reply:
I think we talked about this way back and decided against it, but I don't remember the full rationale. I think maybe we liked that typescript was a single package?